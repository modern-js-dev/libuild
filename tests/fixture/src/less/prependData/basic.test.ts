import { getLibuilderTest } from '@modern-js/libuild-test-toolkit';

describe('fixture:less', function () {
  it('prependData', async () => {
    const bundler = await getLibuilderTest({
      root: __dirname,
      input: {
        basic: './basic.less',
      },
      style: {
        less: {
          prependData: `@base-color: #c6538c;`,
        },
      },
    });
    await bundler.build();
    bundler.expectCSSOutputMatchSnapshot();
  });
});
