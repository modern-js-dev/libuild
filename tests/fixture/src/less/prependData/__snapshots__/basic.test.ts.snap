// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fixture:less prependData 1`] = `
"/* basic.less */
.a {
  color: #c6538c;
}
"
`;
